sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/mapping/CosmosIndexingPolicy.java

```diff
@@ -13,13 +13,21 @@
 import java.lang.annotation.Target;
 
 /**
- * Annotation for cosmos indexing policy
+ * Annotation for cosmos indexing policy.
+ * Using this annotation will overwrite the cosmos indexing policy currently in Azure Portal.
```

Comment:
This is not true by default now with this breaking change. 
We should clarify it here. 

```suggestion
 * Using this annotation will NOT overwrite the cosmos indexing policy currently in Azure Portal by default. See `overwritePolicy()` for more details. 
```